c/tests/test_trees.c

```diff
@@ -1202,6 +1213,14 @@ test_simplest_unary_records(void)
     CU_ASSERT_EQUAL(tsk_treeseq_get_num_trees(&ts), 1);
     CU_ASSERT_EQUAL(tsk_treeseq_get_num_populations(&ts), 1);
 
+    ret = tsk_tree_init(&t, &ts, 0);
+    CU_ASSERT_EQUAL(ret, 0);
+    ret = tsk_tree_first(&t);
+    CU_ASSERT_EQUAL(t.num_children[2], 1);
+    CU_ASSERT_EQUAL(t.num_children[4], 2);
+    CU_ASSERT_EQUAL(t.num_children[5], tsk_tree_get_num_roots(&t));
```

Comment:
This will be a circular test once it's implemented - better to put the actual value in here. Likewise below.

Reply:
Yep. Will change that. Thanks for the comments. Will address them and the Lint error.